### DISCUSSION TITLE
Multiselect max display selected item

### DISCUSSION THREAD

**Discussion by arifrohmanhakim** [mantinedev/mantine#6243] [Feature requests]
Hi,
I really like mantine, thanks for the hard work.
I want to ask, does Mantine support this feature in multiselect components?

![Screenshot 2024-05-16 at 16 30 11](https://github.com/mantinedev/mantine/assets/86143789/90b3498d-aa48-4785-82f3-cb4b9904cc27)

it would be great if multiselect had that feature.
Thanks

**Comment by rtivital** [comment]
https://mantine.dev/combobox/?e=MaxSelectedItems

**Reply by arifrohmanhakim** [reply]
I mean something like this

https://ant.design/~demos/select-demo-responsive/

**Reply by rtivital** [reply]
https://mantine.dev/combobox/?e=MaxDisplayedItems
